se/env.py

```diff
@@ -159,15 +159,26 @@ def validated_ports(ports_str):
         if len(args.slaves) != 1:
             parser.error("Exactly one slave address must be specified for command mode")
 
-    # print out the arguments and option
-    for k,v in sorted(vars(args).iteritems()):
```

Comment:
thanks for the PR!  I think you can achieve your desired compatibility by modifying this line only.  just change `iteritems() -> items()`

Reply:
Indeed changing only that line would work for me (and any Python3 user) but it wouldn't work for any other Python2 user.
This why I suggest to place a version_info check.
Cheers :)

Reply:
My cursory reading of this line says that vars() will return a dict. In this case items() is indeed available on all versions of python